src/directives/profile.js

```diff
@@ -5,7 +5,7 @@
  * Example:
  *
  * <div ngeo-profile="ctrl.profileData"
- *      ngeo-profile-ooptions="ctrl.profileOptions"></div>
+ *      ngeo-profile-options="ctrl.profileOptions"></div>
  *
  * Note: "ctrl.profileOptions" is of type ngeox.profile.ProfileOptions.
```

Comment:
I think we should document the options here (or possibly in `src/d3-ext/profile.js`).